.yarnrc.yml

```diff
@@ -0,0 +1 @@
+nodeLinker: node-modules
```

Comment:
If we do ^^ git ignore changes on `.yarn` part then this `.yarnrc.yml` config should change a little bit more to reflect the localised yarn situation.

For example, it will contain pointer to localised yarn path.
```
yarnPath: .yarn/releases/yarn-4.3.0.cjs
```

See also:
https://github.com/umccr/data-portal-apis/blob/dev/.yarnrc.yml